internal/subshell/bash/bash.go

```diff
@@ -112,7 +106,16 @@ func (v *SubShell) RcFile() (string, error) {
 		return "", errs.Wrap(err, "IO failure")
 	}
 
-	return filepath.Join(homeDir, rcFileName), nil
+	rcFilePath := filepath.Join(homeDir, rcFileName)
+	// On MacOS the .bashrc file is not created by default
+	if runtime.GOOS == "darwin" && !fileutils.FileExists(rcFilePath) {
```

Comment:
I don't think this needs to be macOS only. Also use `TargetExists` instead; since their rc file could be a symlink.